### PR TITLE
added std instance method to pytorch frontend

### PR DESCRIPTION
**Pull request by willsketch**
close https://github.com/unifyai/ivy/issues/14994

### CODE REVIEW

**Comment by willsketch** [comment]
close #14994

File: ivy/functional/frontends/torch/tensor.py [hunk 1]
```diff
@@ -1067,3 +1067,9 @@ def addcdiv(self, tensor1, tensor2, *, value=1):
     @with_unsupported_dtypes({"1.11.0 and below": sign_decorator_dtypes}, "torch")
     def sign(self):
         return torch_frontend.sign(self._ivy_array)
+
+    @with_unsupported_dtypes({"1.11.0 and below": ("float16",)}, "torch")
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/torch/tensor.py:1071]: This decorator should be moved to the `torch_frontend.std` frontend function not this instance method.

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @willsketch 

Thanks for the PR, all changes look good to me😊
Just a small change is needed then we merge the PR🙂

File: ivy/functional/frontends/torch/tensor.py (hunk 1 above)
Comment (willsketch) [reply ivy/functional/frontends/torch/tensor.py]: hey @hirwa-nshuti  Thank you for the review. I worked on the changes you requeted and also worked on the conflicts

Comment (hirwa-nshuti) [reply ivy/functional/frontends/torch/tensor.py]: Now there are other changes that are not intended to would you revert them

Comment (willsketch) [reply ivy/functional/frontends/torch/tensor.py]: Hey @hirwa-nshuti , the other changes i made were to resolve conflicts. Are these the changes you mean

File: ivy/functional/ivy/experimental/manipulation.py [hunk 2]
```diff
@@ -4,7 +4,6 @@
     Tuple,
     Iterable,
     Sequence,
-    Generator,
```
Comment (hirwa-nshuti) [inline ivy/functional/ivy/experimental/manipulation.py:7]: This shouldn't be removed thought

Comment (willsketch) [reply ivy/functional/ivy/experimental/manipulation.py]: hey @hirwa-nshuti i updated my branch, i think all the unnecessary changes should be reverted now!
Thank you for your patience

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 3]
```diff
@@ -8015,6 +8015,42 @@ def test_torch_instance_sign(
     )
 
 
+# std
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="std",
+    dtype_and_x=helpers.dtype_and_values(
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:8023]: This strategy will generate unstable examples you can use this one as it is already implemented to handle statistical values https://github.com/unifyai/ivy/blob/0a32c763fa1d7fd2fd5c9803b31eeb23752e2947/ivy_tests/test_ivy/test_functional/test_core/test_statistical.py#L12

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey,

Just a few changes to make and we merge this item😊

File: ivy/functional/frontends/torch/reduction_ops.py [hunk 4]
```diff
@@ -95,6 +95,7 @@ def median(input, dim=None, keepdim=False, *, out=None):
 
 
 @to_ivy_arrays_and_back
+@with_unsupported_dtypes({"1.11.0 and below": ("float16",)}, "torch")
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/torch/reduction_ops.py:98]:
```suggestion
@with_unsupported_dtypes({"2.0.1 and below": ("float16",)}, "torch")

```

**Review by hirwa-nshuti — APPROVED** [review]
Looks good to me😊
Going to merge✌
Thanks for contributing

**Pull request merged**